frontend/ui/widget/touchmenu.lua

```diff
@@ -1319,7 +1319,7 @@ function TouchMenu:onShowMenuSearch()
                     text = _("Search"),
                     callback = function()
                         local search_for = search_dialog:getInputText()
-                        local status, err = pcall( function() ("test_string"):find(search_for) end)
+                        local status, err = pcall( function() ("test_string"):find(search_for, 1, true) end)
                         if status then
                             search_for = Utf8Proc.lowercase(search_for)
```

Comment:
I think this pcall was actually to test that the input string, if it is a Lua pattern, is a valid pattern, and show "Malformed message" if it is not (if Lua :find() fails using it).
If we don't do pattern find, no need for the `pcall` and the `else` branch, right ?

Reply:
Done.